backend/src/socket/index.js

```diff
@@ -40,6 +41,7 @@ io.on('connection', socket => {
         break;
 
       case EVENTS.CLIENT_ADD_SONG:
+        console.log("Payload is: ", action.payload);
```

Comment:
I think you should remove this line if it is not necessary

Reply:
OK